api/accounts.go

```diff
@@ -351,7 +402,7 @@ func (a *API) electionCountHandler(_ *apirest.APIdata, ctx *httprouter.HTTPConte
 
 // tokenTransfersHandler
```

Comment:
you could take the chance to rename this to tokenTransfersListHandler , for extra consistency